### PR TITLE
Lan manager

### PR DESCRIPTION
**Pull request by atrovato**
### Pull Request check-list

To ensure your Pull Request can be accepted as fast as possible, make sure to review and check all of these items:

- [x] If your changes affects code, did your write the tests?
- [x] Are tests passing? (`npm test` on both front/server)
- [x] Is the linter passing? (`npm run eslint` on both front/server)
- [x] Did you run prettier? (`npm run prettier` on both front/server)
- [x] If you are adding a new features/services, did you run integration comparator? (`npm run compare-translations` on front)
- [x] Did you test this pull request in real life? With real devices? If this development is a big feature or a new service, we recommend that you provide a Docker image to [the community](https://community.gladysassistant.com/) for testing before merging.
- ~~[ ] If your changes modify the API (REST or Node.js), did you modify the API documentation? (Documentation is based on comments in code)~~
- [ ] If you are adding a new features/services which needs explanation, did you modify the user documentation? See [the GitHub repo](https://github.com/GladysAssistant/v4-website) and the [website](https://gladysassistant.com).
- [x] Did you add fake requests data for the demo mode (`front/src/config/demo.js`) so that the demo website is working without a backend? (if needed) See [https://demo.gladysassistant.com](https://demo.gladysassistant.com).

NOTE: these things are not required to open a PR and can be done afterwards / while the PR is open.

### Description of change

LAN Manager is able to detect device en network, and to check their presence.

Some points:
 - The integration picture looks a bit light
 - It doesn't not work 100% on WSL2 (MAC is missing, device is not manageable)
 - It changed the device name in the presence box on the dashboard to display the device name instead of the feature name for the presence device list

Community exchanges: https://community.gladysassistant.com/t/detection-de-presence-via-le-reseau-local/6131

### CODE REVIEW

File: server/services/lan-manager/lib/lan-manager.getDiscoveredDevices.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
+/**
+ * @description Get all found preipherals as devices.
```
Comment (Pierre-Gilles) [inline server/services/lan-manager/lib/lan-manager.getDiscoveredDevices.js]: typo preipherals

**Review by Pierre-Gilles — COMMENTED** [review]
Nice PR! I don't have much feedbacks on the code itself, just one feedback on the UI.

Could this design be a bit improved? The "+" button is a bit far on the right it's weird?

Screenshot 2023-03-24 at 13 19 40

Also, could you add a little message to explain for beginners what is a CIDR and how to fill this part ?

**Comment by atrovato** [comment]
A little message is a bit hard... it is not so little
I also moved the "+" button, but I think that now it is "weird" to get the "+" not under the "delete".
Should I move "+" and "delete" on first column?

image

**Comment by Pierre-Gilles** [comment]
@atrovato Still it's way better like this :) Thanks for changing 🙏

Are you still waiting for feedbacks on the PR or should this be merged to prod ?

**Comment by atrovato** [comment]
I already have some good feedback from the community, I also have some bad, but I'm not able to reproduce it.
I think we can merge it, it will be usable for most, and trace and fix exotic issues later.

**Comment by Pierre-Gilles** [comment]
Ok thanks @atrovato, I'll merge this now to deploy in next release!

**Pull request merged**